events/break-free/index.html

```diff
@@ -72,6 +72,8 @@
             {{ description }}
           </div>
         </script>
+
+  <script type="text/javascript" src="js/action-kit-service.js"></script>

```

Comment:
Gotcha - that makes sense.  Let's leave it as is, then.  I'll file a new issue about bundling / dependency management.